lms/static/scripts/frontend_apps/components/test/BasicLtiLaunchApp-test.js

```diff
@@ -500,6 +502,20 @@ describe('BasicLTILaunchApp', () => {
       renderLTILaunchApp();
       assert.isTrue(fakeApiCall.notCalled);
     });
+
+    it('reports a submission if VitalSource configuration is present, even if `contentUrl` not set', async () => {
```

Comment:
```suggestion
    it('reports a submission if VitalSource configuration is present, even if `viaUrl` not set', async () => {
```

The key in the configuration is `viaUrl`. `contentURL` is the internal state in the component.